middleware/recovery.go

```diff
@@ -78,6 +78,13 @@ func (r *bufferedReadCloser) Close() error {
 	return r.reader.Close()
 }
 
+// RemovedHeaders defines a list of HTTP headers that will be redacted from the
+// request in the Recovery handler--if any logging or other output occurs, these
+// headings will have value '<redacted>'.
+var RemovedHeaders []string
```

Comment:
Should we default this to `[]string{"Authorization"}` as that header is always going to contain sensitive information?

Reply:
Agreed! Fixed!